extensions/healthie/actions/createMetricEntry/config/fields.ts

```diff
@@ -0,0 +1,25 @@
+import { type Field, FieldType } from '@awell-health/extensions-core'
+
+export const fields = {
+  userId: {
+    id: 'userId',
+    label: 'User ID',
+    description: 'The ID of the patient that this entry should be attached to',
+    type: FieldType.STRING,
+    required: true,
+  },
+  category: {
+    id: 'category',
+    label: 'Category',
+    description: 'Specifies what kind of metric we are storing',
+    type: FieldType.STRING,
+    required: true,
+  },
+  metricStat: {
+    id: 'metricStat',
+    label: 'Metric stat',
+    description: 'The actual data value for the metric',
+    type: FieldType.STRING,
```

Comment:
This comment is still unaddressed: https://github.com/awell-health/awell-extensions/pull/272#discussion_r1350272443

Reply:
@kubiak85 

> Looking at their API, it seems to have been designed to handle strings, so it's a good idea to stick with that convention

The thing is that in the Awell platform, all metric-related data points will be of type `number`. This basically means our users will not be able to choose any numeric data point for this action field because it's of value type string (our system is strongly typed).

That's why we it makes more sense to make the action field a numeric value type and just make sure we stringify the value before sending it over to Healthie. So please do make the change and adjust the action accordingly.

> Unless we have someone on the team who will be responsible for handling updates for this extension?

I don't understand how this change would affect handling updates for this extension. Parsing a number to a string is pretty straightforward and I don't see any risk really for future development.